integrations/pom.xml

```diff
@@ -261,6 +261,25 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>gitea</artifactId>
+      <version>1.4.0--rc182.9eb947470fcf</version>
```

Comment:
looks like broken naming? shouldn't be a double dash

Reply:
Unfortunately, that's correct for this specific version. We had some issues with the automatic release. The next one does not have such suffix (and the double dash).

Reply:
shall we wait till fixed? to get a proper version, having rc in here is off too

Reply:
Sure. Let's wait till next release.